games-fps/alephone/alephone_marathoninfinity-20150907.recipe

```diff
@@ -77,9 +71,9 @@ INSTALL()
 	ALEPHONE="`finddir B_SYSTEM_BIN_DIRECTORY`/alephone"
 	DATA_PATH="`finddir B_SYSTEM_DATA_DIRECTORY`/AlephOne/Marathon Infinity"
 
-cat >"$appsDir/Marathon_Infinity" <<EOF
+	cat >"$appsDir/Marathon_Infinity" <<EOF
 #!/bin/sh
-exec $ALEPHONE "$DATA_PATH $1"
+exec $ALEPHONE "$DATA_PATH" "$@"
```

Comment:
Sorry, my mistake. I tested my suggestion with `cat` instead of `cat <<EOF`, and it turns out we need to protect the `$`:
```diff
-exec $ALEPHONE "$DATA_PATH" "$@"
+exec $ALEPHONE "$DATA_PATH" "\$@"
```
otherwise the last line of the script will end with `""` instead of `"$@"`.

Everything else looks OK.
Hmm, are you sure the 3 `CHECKSUM_SHA256` are OK?